### ISSUE TITLE
Suggest error detail fields as named arguments in an error constructor expression

### ISSUE THREAD

**Issue by MaryamZi** [ballerina-platform/ballerina-lang#29705]
**Description:**
Creating a new issue for IV of https://github.com/ballerina-platform/ballerina-lang/issues/24382.

When a record is specified as the detail type of a user-defined error, can we suggest the names of the fields in the detail as the named args in the error constructor?

**Comment by malinthar** [comment]
Blocked by https://github.com/ballerina-platform/ballerina-lang/issues/32755
